.github/workflows/gcp_image.yml

```
@@ -1,11 +1,35 @@
-name: Build and test image for GCP
+name: Build and test image for GCP SEV
```

```suggestion
name: Build and test image for GCP SEV and TDX
```
it is not only SEV right?